README.md

```diff
@@ -187,6 +187,20 @@ GitHub github = Feign.builder()
                      .target(GitHub.class, "https://api.github.com");
 ```
 
+If you need to pre-process the response before give it to the Decoder, you can use the [InterceptorDecoder](/feign/codec/InterceptorDecoder.java).
+
+```java
+GitHub github = Feign.builder()
+                     .decoder(
+                         new InterceptorDecoder(
```

Comment:
It is ok for me! But for make this Decoder private (at least package private) I have to move it from the ```feign.codec```package to the ```feign```one... is that ok for you?